packages/framework/aqueduct/src/data-object-factories/pureDataObjectFactory.ts

```diff
@@ -208,17 +208,22 @@ export class PureDataObjectFactory<
 	 * @param context - The context being used to create the runtime
 	 * (the created object will have its own new context created as well)
 	 * @param initialState - The initial state to provide to the created data store.
+	 * @param groupId - group to which this data stores belongs to. The datastores with groupId might not be loaded
```

Comment:
What does it mean for the caller to not provide this? Can we document that?

Reply:
sure.